src/annotator/components/Toolbar.tsx

```diff
@@ -84,7 +84,7 @@ export type ToolbarProps = {
   /**
    * Callback for a button that creates an annotation.
    */
-  createAnnotation: (tool: AnnotationTool) => void;
+  createAnnotation: (tool: AnnotationTool | null) => void;
```

Comment:
Re-using the existing `createAnnotation` method with a `null` argument avoids adding an extra method and adding the plumbing and tests to get the signal from the toolbar to the Guest. We might need to split this into two separate methods in future if this becomes awkward.

Reply:
This looks like a pragmatic and obvious enough way to support toggling.